src/internal/pfsdb/projects.go

```diff
@@ -37,8 +37,7 @@ func (err *ProjectNotFoundError) Error() string {
 }
 
 func (err *ProjectNotFoundError) Is(other error) bool {
-	_, ok := other.(*ProjectNotFoundError)
-	return ok
+	return errors.As(other, &ProjectNotFoundError{})
```

Comment:
Check if you need the double pointer here.

Reply:
This is actually only one pointer:
``` go
// ProjectNotFoundError is returned by GetProject() when a project is not found in postgres.
type ProjectNotFoundError struct {
	Name string
	ID   ProjectID
}
```

But to confirm, that this works, we're using it in `projects_test.go`
```
...
		// validate error for attempting to get non-existent project.
		_, err = pfsdb.GetProject(cbCtx, tx, 3)
		require.YesError(t, err, "should not be able to get non-existent project")
		require.True(t, (&pfsdb.ProjectNotFoundError{ID: 3}).Is(err))
		return nil
	}))
```